beancount_periodic/common/config.py

```diff
@@ -7,7 +7,10 @@
 
 from dateutil.relativedelta import relativedelta
 
-from beancount.utils.date_utils import parse_date_liberally
+try:
+    from beancount.utils.date_utils import parse_date_liberally
+except ImportError:
+    from beangulp.date_utils import parse_date_liberally
```

Comment:
```suggestion
    from beanprice.date_utils import parse_date_liberally
```

(also, hi, long time not see :)

Reply:
I've tested this works with https://github.com/autrilla/beancount-periodic/

Reply:
Howdy! Long time no see indeed!